name: train
run_id: commands[0]
env COMSPEC: C:\WINDOWS\system32\cmd.exe
env NUMBER_OF_PROCESSORS: 16
env PATHEXT: .COM;.EXE;.BAT;.CMD;.VBS;.VBE;.JS;.JSE;.WSF;.WSH;.MSC;.PY;.PYW;.CPL
env PROCESSOR_ARCHITECTURE: AMD64
env PROGRAMDATA: C:\ProgramData
env PROGRAMFILES: C:\Program Files
env PROGRAMFILES(X86): C:\Program Files (x86)
env SYSTEMDRIVE: C:
env SYSTEMROOT: C:\WINDOWS
env TEMP: C:\Users\harsh\AppData\Local\Temp
env TMP: C:\Users\harsh\AppData\Local\Temp
env USERPROFILE: C:\Users\harsh
env LANG: en_US.UTF-8
env PATH: C:\Users\harsh\gitdir\MLOps\Assignment2\.tox\test_package\Scripts;C:\Python311\Scripts\;C:\Python311\;C:\Program Files\Eclipse Adoptium\jdk-17.0.1.12-hotspot\bin;C:\Windows\system32;C:\Windows;C:\Windows\System32\Wbem;C:\Windows\System32\WindowsPowerShell\v1.0\;C:\Windows\System32\OpenSSH\;C:\Program Files (x86)\NVIDIA Corporation\PhysX\Common;C:\Program Files\NVIDIA Corporation\NVIDIA NvDLISR;C:\WINDOWS\system32;C:\WINDOWS;C:\WINDOWS\System32\Wbem;C:\WINDOWS\System32\WindowsPowerShell\v1.0\;C:\WINDOWS\System32\OpenSSH\;C:\Program Files\Amazon\AWSCLIV2\;C:\Program Files\Docker\Docker\resources\bin;C:\ProgramData\DockerDesktop\version-bin;C:\Program Files\nodejs\;C:\ProgramData\chocolatey\bin;C:\Program Files\Git\cmd;C:\Program Files\GitHub CLI\;C:\Users\harsh\AppData\Local\Microsoft\WindowsApps;C:\Users\harsh\AppData\Local\Programs\Microsoft VS Code\bin;C:\Users\harsh\AppData\Local\atom\bin;C:\Users\harsh\AppData\Roaming\npm
env PIP_DISABLE_PIP_VERSION_CHECK: 1
env PYTHONIOENCODING: utf-8
env PYTHONPATH: .
env PYTHONHASHSEED: 0
env TOX_ENV_NAME: train
env TOX_WORK_DIR: C:\Users\harsh\gitdir\MLOps\Assignment2\.tox
env TOX_ENV_DIR: C:\Users\harsh\gitdir\MLOps\Assignment2\.tox\test_package
env VIRTUAL_ENV: C:\Users\harsh\gitdir\MLOps\Assignment2\.tox\test_package
env COLUMNS: 165
env LINES: 20
metadata pid: 26024
cwd: C:\Users\harsh\gitdir\MLOps\Assignment2
allow: C:\Users\harsh\gitdir\MLOps\Assignment2\.tox\test_package\Scripts\*
cmd: python classification_model/train_pipeline.py
exit_code: 1

standard error:
Traceback (most recent call last):
  File "C:\Users\harsh\gitdir\MLOps\Assignment2\classification_model\train_pipeline.py", line 2, in <module>
    from config.core import config
  File "C:\Users\harsh\gitdir\MLOps\Assignment2\classification_model\config\core.py", line 7, in <module>
    import classification_model
  File "C:\Users\harsh\gitdir\MLOps\Assignment2\classification_model\__init__.py", line 3, in <module>
    from classification_model.config.core import PACKAGE_ROOT, config
  File "C:\Users\harsh\gitdir\MLOps\Assignment2\classification_model\config\core.py", line 88, in <module>
    config = create_and_validate_config()
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\harsh\gitdir\MLOps\Assignment2\classification_model\config\core.py", line 82, in create_and_validate_config
    model_config=ModelConfig(**parsed_config.data),
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "pydantic\main.py", line 342, in pydantic.main.BaseModel.__init__
pydantic.error_wrappers.ValidationError: 1 validation error for ModelConfig
test_size
  field required (type=value_error.missing)
